Translation corrections (#105)
07th-mod/higurashi-console-arcs: Update/some_008.txt

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "私と千紗登ちゃんは車を降りると、居並ぶ人たちに声をかける。が、ほとんどが騒ぎを聞きつけて集まった人らしく、要領の得ない表情で首を横に振るだけだった。",

> 				NULL, "<size=-2>Chisato-chan and I tried asking some of the nearby onlookers after getting got out of the car. However, most of them seemed to have been drawn by the noise, so they simply shook their heads looking just as clueless as us.", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>Chisato-chan and I tried asking some of the nearby onlookers after getting out of the car. However, most of them seemed to have been drawn by the noise, so they simply shook their heads looking just as clueless as us.", Line_Normal);
```

	} else {
		OutputLine(NULL, "私と千紗登ちゃんは車を降りると、居並ぶ人たちに声をかける。が、ほとんどが騒ぎを聞きつけて集まった人らしく、要領の得ない表情で首を横に振るだけだった。",

> 				NULL, "Chisato-chan and I tried asking some of the nearby onlookers after getting got out of the car. However, most of them seemed to have been drawn by the noise, so they simply shook their heads looking just as clueless as us.", Line_Normal);

Reviewer: ```suggestion
				NULL, "Chisato-chan and I tried asking some of the nearby onlookers after getting out of the car. However, most of them seemed to have been drawn by the noise, so they simply shook their heads looking just as clueless as us.", Line_Normal);
```

//r見るとその人は、介護士さんだった。あまり親しくなかったので名前は知らないけど、時々シフトが一緒になって、挨拶くらいは交わして顔は覚えている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "見るとその人は、介護士さんだった。あまり親しくなかったので名前は知らないけど、時々シフトが一緒になって、挨拶くらいは交わして顔は覚えている。",

> 			NULL, "It was one of the caregivers. I didn't know her name because we were never very close, but we occasionally had shifts together. At the very least, I knew her enough to recognize her face and say hi to her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was one of the caregivers. I didn't know her name because we were never very close, but we occasionally had shifts together. At the very least, I knew her well enough to say hi to her when I recognized her face.", Line_Normal);
```

//r呆然と。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "呆然と。",

> 			NULL, "I couldn't believe what I'd heardy.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I couldn't believe what I'd heard.", Line_Normal);
```
